src/Platforms/Gcc/UtestPlatform.cpp

```diff
@@ -72,8 +73,13 @@ static void GccCygwinPlatformSpecificRunTestInASeperateProcess(UtestShell* shell
     cpid = PlatformSpecificFork();
 
     if (cpid == -1) {
-        result->addFailure(TestFailure(shell, "Call to fork() failed"));
-        return;
+        if(errno == EINTR) { /* Retry once if interrupted */
```

Comment:
Using `goto` would be really neat here -- but only if we knew for sure that the fork would eventually succeed...